app/lib/ui/widgets/atr_preview.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:app/core/models/animal_transport_record.dart';
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+
+class AnimalTransportRecordPreview extends StatelessWidget {
+  final AnimalTransportRecord atr;
+
+  AnimalTransportRecordPreview({@required this.atr});
+
+  @override
+  Widget build(BuildContext context) {
+    return Card(
+      child: Material(
+        child: ListTile(
+          leading: Icon(Icons.local_shipping),
+          title: Text(
+              'Delivery for ${atr.deliveryInfo.recInfo.receiverCompanyName}'),
+          subtitle: Text(
+              'Date: ${DateFormat("yyyy-MM-dd hh:mm").format(atr.vehicleInfo.dateAndTimeLoaded)}, Animals: ${atr.vehicleInfo.animalSpeciesLoaded().join(',')}'),
```

Comment:
- [ ] Now that I look at it... we can just drop `Date: , Animals` because it is obvious from output.